sdk/translation/Azure.AI.Translation.Document/tests/samples/Sample_MultipleInputs.cs

```diff
@@ -44,36 +44,7 @@ public void MultipleInputs()
                     input2
                 };
 
-            DocumentTranslationOperation operation = client.StartTranslation(inputs);
```

Comment:
The idea was to show that the user can manually poll for sync scenarios to keep all their code always sync.
Now, they don't need to manually poll, and they can still keep their code sync. I like this!